modules/flyte/acr.tf

```diff
@@ -0,0 +1,11 @@
+resource "azurerm_role_assignment" "flyte_controlplane_acr" {
+  scope                = var.azurerm_container_registry_id
+  role_definition_name = "AcrPull"
```

Comment:
What does the `AcrPull` `role_definition_name` do?

Reply:
I'm not sure if that will be necessary for flyte...my understanding is that we will pull images from some registry within the cluster using the `domino-quay-repos` imagePullSecret. In my mind this section would map to [this](https://github.com/dominodatalab/terraform-aws-eks/blob/4aa12b9ba675474c0ad8397a0cf41575cb4e96cd/modules/flyte/iam.tf#L33) eks terraform where I allow putting and getting from s3 buckets.